cmd/kubelet/app/options/options.go

```diff
@@ -390,6 +401,8 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.BootstrapCheckpointPath, "bootstrap-checkpoint-path", f.BootstrapCheckpointPath, "<Warning: Alpha feature> Path to to the directory where the checkpoints are stored")
 
 	// DEPRECATED FLAGS
+	fs.StringVar(&f.BootstrapKubeconfig, "experimental-bootstrap-kubeconfig", f.BootstrapKubeconfig, "deprecated: use --bootstrap-kubeconfig")
```

Comment:
This is going to result in some funny help text, something like:
```
--experminetal-bootstrap-kubeconfig string  deprecated: use --bootstrap-kubeconfig (DEPRECATED: Use --bootstrap-kubeconfig) (default: $f.BootstrapKubeconfig)
```

Reply:
updated